apps/flutter_exercise/lib/views/widgets/login_widgets/form.dart

```diff
@@ -0,0 +1,231 @@
+// Create a Form widget.
+import 'package:flutter/material.dart';
+import 'package:flutter_application_1/views/widgets/login_widgets/checkbox.dart';
+
+class LoginForm extends StatefulWidget {
+  const LoginForm({super.key});
+
+  @override
+  MyCustomFormState createState() {
+    return MyCustomFormState();
+  }
+}
+
+// Create a corresponding State class.
+// This class holds data related to the form.
+class MyCustomFormState extends State<LoginForm> {
+  // Create a global key that uniquely identifies the Form widget
+  // and allows validation of the form.
+  //
+  // Note: This is a GlobalKey<FormState>,
+  // not a GlobalKey<MyCustomFormState>.
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    // Build a Form widget using the _formKey created above.
+    return Container(
+      margin: const EdgeInsets.only(left: 40, right: 40, top: 100),
+      child: Form(
+        key: _formKey,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            const Align(
+              alignment: Alignment.topRight,
+              child: Text(
+                "Ayuda",
+                textAlign: TextAlign.right,
+                style: TextStyle(
+                    color: Color.fromARGB(255, 23, 57, 25),
+                    decoration: TextDecoration.underline),
+              ),
+            ),
+            Container(
+              margin: const EdgeInsets.only(left: 70, right: 70, bottom: 20),
+              child: const Column(
+                children: [
+                  Text(
+                    "Regístrate",
+                    style: TextStyle(
+                      color: Colors.orangeAccent,
+                      fontWeight: FontWeight.bold,
+                      fontSize: 32,
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(top: 10),
+                  ),
+                  Text("Pon el marcapáginas, solo nos llevará unos segundos",
+                      textAlign: TextAlign.center,
+                      style: TextStyle(
+                        color: Color.fromARGB(255, 23, 57, 25),
+                        fontWeight: FontWeight.bold,
+                      ))
+                ],
+              ),
+            ),
+            _fieldForm("nombre"),
+            _fieldForm("email"),
+            _fieldForm("contraseña"),
+            const Align(
```

Comment:
@Mazit0 Why repeat the subtitle two times, you should refactor in a method, or add a new parameter to _fieldForm method to allow add the subtitle

Reply:
esto lo hice después de la pr xD